site/content/api/Command-And-Control-API.md

```diff
@@ -12,43 +12,39 @@ THIS IS A FIRST DRAFT VERSION OF THE API SPEC AND THUS STILL SUBJECT TO DISCUSSI
 IF YOU THINK THAT THE SPEC SHOULD BE CHANGED OR CAN BE IMPROVED PLEASE DO NOT HESITATE TO OPEN AN ISSUE OR DROP US A LINE ON THE MAILING LIST.
 {{% /warning %}}
 
-A client can be a device connected to Hono directly (using an AMQP connection) or via a *Protocol Adapter* (for example via MQTT,
-LWM2M, etc). Devices and *Protocol Adapters* can also poll Hono periodically in order check for new command & control messages without
-maintaining a constant connection to Hono infrastructure.
+The Command & Control API is defined by means of AMQP 1.0 message exchanges, i.e. a client needs to connect to Hono using AMQP 1.0 in order to invoke operations of the API as described in the following sections. Throughout the remainder of this page we will simply use AMQP when referring to AMQP 1.0.
 
-The Command & Control API is defined by means of AMQP 1.0 message exchanges, i.e. a client needs to connect to Hono using an AMQP 1.0 client in 
-order to invoke operations of the API as described in the following sections.
+# Operations
 
-# Southbound Operations
+The following API defines operations that can be used by *Business Applications* to send control messages to *Protocol Adapters* (and by *Protocol Adapters* to receive them) as well as the response from the *Protocol Adapters* back to the *Business Applications*.
```

Comment:
IMHO a *Business Application* uses this API to send a command to a device. An application should not need to know about the fact, that a thing as a *Protocol Adapter* exists, should it?

It is a little tricky to find the right scope here. My current feeling is that we do not have a southbound C & C API at all but that the involvement of the protocol adapters with the AMQP Network is merely an implementation detail. In the same spirit, we also do not define the northbound operations of the Event and Telemetry APIs as an interaction between a consuming application and the AMQP Network, do we?